### ISSUE TITLE
Maxwell daemon to publish to kafka with secure SSL

### ISSUE THREAD

**Issue by RMizzi** [zendesk/maxwell#1336]
Is it possible to configure the maxwell daemon to publish messages to kafka using SSL? -- I cannot find any documentation related to this

**Comment by sintezcs** [comment]
Any updates on this topic? I am also interested in this question.

**Comment by RMizzi** [comment]
Hi @sintezcs , we have managed to do this succesfully since all kafka configuration settings are supported when setting up the maxwell daemon producer. I'm afraid i cannot give you the exact set up details since I didnt do it myself but we have confirmed that it is definitely supported.

**Comment by osheroff** [comment]
agreed, please see https://docs.confluent.io/current/kafka/authentication_ssl.html for details on how to set this up.  the only thing different is that when you configure maxwell's kafka settings in config.properties you will need to prefix all the kafka settings with `kafka.`

**Comment by dbhatia2** [comment]
@RMizzi I  am same boat. Can you please share you experiences setting up maxwell kafka using SSL? It would great if you can share what settings worked for you?

**Comment by codermannn** [comment]
I have configured Maxwell v1.40.5 (OpenJDK 11) to connect to an AWS MSK cluster using SSL. I’ve added the following settings in the Maxwell configuration file to use the default JVM truststore (cacerts):

```
kafka.security.protocol=SSL
kafka.ssl.truststore.location=/usr/local/openjdk-11/lib/security/cacerts
kafka.ssl.truststore.password=changeit
```

Despite this configuration, Maxwell fails with an error stating that it cannot find the topic and times out. I have verified that writes to the MSK cluster are successful using other applications that use a Java client, so the MSK cluster itself seems to be setup correctly.

I’m not sure what else I might be missing in the configuration.

The MSK is on version 3.5.1.
I have set the kafka_version=3.4.0 in the Maxwell config as 3.5.1 is not still supported by the version.
